apps/client/src/app/services/helper.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { NgForm } from '@angular/forms';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class HelperService {
```

Comment:
I think HelperService is one of those really meaningless namings :D helper for what?